%YAML:1.0

leftCam_fx: 762.725
leftCam_fy: 762.725
leftCam_cx: 640.5
leftCam_cy: 360.5

rightCam_fx: 762.725
rightCam_fy: 762.725
rightCam_cx: 640.5
rightCam_cy: 360.5

t: [-0.5, 0, 0]
R: [1, 0, 0, 0, 1, 0, 0, 0, 1]

# for viewer
window_width: 1280
window_height: 720

viewer_point_x: 0
viewer_point_y: 0
viewer_point_z: -1
viewer_point_f: 762.725

# for drawing
goal_height: 2
goal_length: 3
goal_width1: 1.5
goal_width2: 1

# blob parameters
blob_minThres: 10
blob_maxThres: 100
blob_minArea: 100
blob_maxArea: 1000000
blob_minCircularity: 0.7

# Line method parameters
line_roi_size: 50

# HoughLine parameters
Hough_minLength: 20